_events/9.md

```diff
@@ -0,0 +1,41 @@
+---
+title:  "Meeting #5 🌳"
```

Comment:
remove the tree emoji, this was climate strike specific and seems out of place here, what do you think?

Reply:
done.

Reply:
not sure how to add changed to a pull request that has been accepted, so i created a new pull request... :)